### PR TITLE
common elements in lists

### PR DESCRIPTION
**Pull request by Anuhyadasaraju**

### CODE REVIEW

File: half_pyramid_pattern.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# VERSION  DATE             AUTHOR           CHANGES
+# -------  ----             -------          -------
+# 1.0      29th Mar, 2024   Anuhya Dasaraju  Initial version
+#
+# ------------------------------------------------------------------------
+
+value = 1
+
+for i in range(0, 5):
+    for j in range(0, i+1):
```
Comment (vbhaskarvrk) [inline half_pyramid_pattern.py:15]: Please do not use i,j, k kind of variables. Use meaningful names. later.

File: lists_common_elements.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+# AUTHOR  : Anuhya Dasaraju
+# VERSION  DATE             AUTHOR           CHANGES
+# -------  ----             -------          -------
+# 1.0      29th Mar, 2024   Anuhya Dasaraju  Initial version
+#
+# ------------------------------------------------------------------------
+def commom_elements(l1, l2):
+    common_ele = []
+    for i in list1:
```
Comment (vbhaskarvrk) [inline lists_common_elements.py:13]: i -> list1_item
j -> list2_item

use meaningful variable names.

File: max_element_in_list.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+# -------  ----             -------          -------
+# 1.0      29th Mar, 2024   Anuhya Dasaraju  Initial version
+#
+# ------------------------------------------------------------------------
+def largest_element(list):
+    max = 0
+    for i in list:
+        if i > max:
+            max = i
+
```
Comment (vbhaskarvrk) [inline max_element_in_list.py:16]: Don;t leave blank lines between valid codes. variables names need fixes.

**Review by vbhaskarvrk — APPROVED** [review]

File: half_pyramid_pattern.py (hunk 1 above)
Comment (Anuhyadasaraju) [reply half_pyramid_pattern.py:15]: okay bhaskar

File: max_element_in_list.py (hunk 3 above)
Comment (Anuhyadasaraju) [reply max_element_in_list.py:16]: sure will take care from next time

**Pull request merged**